other_recipes/fried rice.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+   <h1>Fried Rice</h1>
+   <img src="American_fried_rice.jpg"width="30%">
+   <h1>Description</h1>
+   <p> Fried rice is a dish of cooked rice that has been stir-fried in a wok or a frying pan and is usually mixed with other ingredients</p>
+   <h1>Ingredients</h1>
+   <li>2 cups cooked rice</li>
+   <li>1/2 cup chicken stock</li>
+   <li>3 tbsp oil for frying</li>
+   <li>1 cup onion diced</li>
+   <li>1 cup mixed Vegetables Carrots, Sweet Peas, Sweet Corn and green beans</li>
+   <li>1/2 tsp thyme</li>
+   <li>1 tsp Curry powder</li>
+   <li>2 scallions diced</li>
+   <li>1 chicken stock cube</li>
+   <li>1 cup beef liver cubed</li>
+   <li>salt to taste</li>
+   <li>1/4 tsp Cayenne pepper</li>
```

Comment:
- [ ] Please wrap your `li` tag in a `ul` elements. Read about the `li[` element to learn how to use them.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li)